ppr-ui/src/utils/roles-helper.ts

```diff
@@ -0,0 +1,15 @@
+export function getDescriptiveUserRole (authRoles: Array<string>): string {
+  const accessRole = []
+
+  if (authRoles.includes('ppr_staff')) return 'STAFF_PPR'
+  if (authRoles.includes('mhr_staff')) return 'STAFF_MHR'
+  if (authRoles.includes('staff') && authRoles.includes('helpdesk')) return 'STAFF_PPR'
```

Comment:
Not sure about this combination. But I saw it in other tests, so added it here. @cameron-freshworks